app/javascript/registration/register.js

```diff
@@ -26,6 +26,14 @@ export default function DorRegistration(initOpts) {
       return document.querySelector('[data-rcparam="workflowId"]').value
     },
 
+    contentType : function() {
+      return document.querySelector('[data-rcparam="contentType"]').value
```

Comment:
Do we need to have a special rcparam? Can we just use the `[data-content-type-target='contentType']`?

Reply:
yeah, we can ditch that (also on the viewing direction element) and just use the data-content-type-target